README.md

```diff
@@ -18,39 +18,121 @@ based on the Open Cluster Management project.
+the Red Hat Advanced Cluster Management operator and which will serve as the central ACM hub.
+We also assume you have admin access to other OpenShift cluster(s) that you will import to the ACM hub cluster,
+these will play the role of near edge location(s) managed from the central ACM hub.
+
+#### ACM setup
+
+The following setup steps need to be done as an admin user with `cluster-admins` privileges.
+We will describe how to do them using the OpenShift Console but there may be other ways to achieve the same results.
+
+1. On the ACM hub cluster, install the Advanced Cluster Management for Kubernetes operator.
+   In the OpenShift Console of the cluster, find and install the operator
+   in Menu > Operators > OperatorHub.
+1. Create the MultiClusterHub.
+   In the OpenShift Console in Menu > Operators > Installed Operators > Advanced Cluster Management for Kubernetes >
+   the MultiClusterHub tab, use the "Create MultiClusterHub" button to create its configuration.
+1. Refresh the OpenShift Console page.
+1. Create a cluster set for your near edge clusters.
+   In the OpenShift Console this can be done in All Clusters menu > Infrastructure > Clusters > Cluster sets tab.
+   This repository assumes the name `poc-near-edge` in `clusterSets`
+   in [acm/registration/near-edge/base/near-edge.yaml](acm/registration/near-edge/base/near-edge.yaml)
+   so update that YAML file if you use a different name.
+1. Import the near edge cluster(s).
+   In the OpenShift Console this can be done in All Clusters menu > Infrastructure > Clusters > Cluster list tab.
+   In the Cluster set popup menu, select the cluster set you created for the near edge clusters.
+   If you use the Import mode of "Enter your server URL and API token for the existing cluster",
+   obtain the address of the server and the token on the other cluster's OpenShift Console 
+   at the top-right corner user menu > Copy login command, for an admin user.
+
+#### Create namespaces for applications
+
+For each of the deployed models-turned-into-application, we will create separate namespace/project on the ACM hub cluster.
+The ACM setup in this repository expects namespaces `tensorflow-housing-app` and `bike-rental-app`.
+
+As a regular user in the OpenShift Console, in the local-cluster > Developer menu, Project popup, use the "Create Project" button;
+alternatively, run
+```bash
+oc new-project tensorflow-housing-app
+oc new-project bike-rental-app
+```
+
+#### Bind namespaces to the cluster set
+
+The newly created namespaces need to be bound to the cluster set that contains the near edge clusters.
+
+On the ACM hub cluster, in the OpenShift Console in the All Clusters menu > Infrastructure > Clusters > Cluster sets tab,
+click the three-dot button at the right of the cluster set, select Edit namespace bindings, and add the namespaces you've created.
+
+This is the last step that needs to be done as an admin user with `cluster-admins` privileges.
```

Comment:
```suggestion
This is the last step that needs to be done as an admin user with `cluster-admin` privileges.
```

Reply:
Applied in 171844e691d75ec81b322b98817751b7dd7097a6.